.github/workflows/forest.yml

```diff
@@ -225,4 +225,4 @@ jobs:
         run: |
           chmod +x ~/.cargo/bin/forest*
       - name: Snapshot export check
-        run: ./scripts/tests/calibnet_export_check.sh
+        run:
```

Comment:
Did you mean to remove this?

Reply:
Nope, I think I just cut it instead of copying.